src/pages/Portfolio/ImpactDashboard/EducationModule.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+	<async-portfolio-section @visible="whenVisible" data-testid="credit-stats">
+		<kv-grid class="tw-grid-cols-12 tw-items-center">
+			<div class="tw-col-span-12 lg:tw-col-span-6">
+				<div class="tw-flex tw-rounded tw-h-12 tw-w-12 tw-bg-primary tw-p-1 tw-mr-2">
+					<img class="tw-w-10 tw-h-10" alt="Leaf heart" :src="imageRequire(`./leaf_heart.svg`)">
+				</div>
+				<h2 class="tw-mb-0.5">
+					Donations power progress
+				</h2>
+				<p class="tw-text-subhead">
+					<!-- eslint-disable-next-line max-len -->
+					Every dollar donated to Kiva helps bring us one step closer to financial access for all. See what we’ve achieved with your support.
+				</p>
+			</div>
+			<div class="tw-col-span-12 lg:tw-col-span-6">
+				<education-post :post="post" :loading="!loadingPromise" />
```

Comment:
Why is this looking for !loadingPromise? That seems like the loading animation will be displayed when the post isn't loading